security-rules.bolt

```diff
@@ -137,12 +141,12 @@ path /landsraad/votingRights {
 
 path /landsraad/questions {
   read() { isSignedIn() }
-  write() { isSwiss() }
+  write() { isSignedIn() }
 }
 
 path /landsraad/answers {
   read() { isSignedIn() }
-  write() { isSwiss() }
+  write() { isSignedIn() }
```

Comment:
Why is this changing?

Reply:
Because with the question the answers are defined too (YES / NO) and currently it is done in the code which runs under non-admin user. 
But of course with the new approach ("set decree according to votingRights") won't be needed. So I will remove it when refactoring.

Reply:
Fixed.